doc/args.rst

```diff
@@ -17,6 +17,21 @@ All command-line parameters, general and module-related, are passed to this main
     --version : @replace
         Display product name & version instead of launching.
 
+    --steps-filter -f : @replace
+        | Allows to filter which steps to execute during launch.
+         String value representing single filter or a set of filters separated by '**:**'.
+         To define exclude pattern use '**!**' symbol at the beginning of pattern.
+        |
+        | A universum step match specified pattern when 'filter' is a substring of step 'name'.
+         This functionality is similar to 'boosttest' and 'gtest' filtering.
+         Except using of special characters like '*', '?', etc. which are ignored.
```

Comment:
This functionality is similar to 'boosttest' and 'gtest' filtering,  except special characters (like '*', '?', etc.) are ignored.